rb/spec/integration/selenium/webdriver/bidi/browsing_context_spec.rb

```diff
@@ -23,6 +23,8 @@ module Selenium
   module WebDriver
     class BiDi
       describe BrowsingContext, only: {browser: %i[chrome edge firefox]} do
+        let(:magic_number) { 'JVBER' }
```

Comment:
What is this magic number?

Reply:
We use it in our current print tests. I thought you knew what it was. I must have copied it from Java. /shrug

Reply:
_Magic number_ is the static base64 encoded bytes which appear at the start of files. For PDF it's value is 'JVBER'. That's why we use it to test here.

Reply:
TIL